spread.yaml

```diff
@@ -576,6 +578,12 @@ prepare: |
     type MATCH | tail -n +2 > "$TESTSLIB"/spread-funcs.sh
     type REBOOT | tail -n +2 >> "$TESTSLIB"/spread-funcs.sh
 
+    # no need to modify anything further for autopkgtest
+    # we want to run as pristine as possible
+    if [ "$SPREAD_BACKEND" = autopkgtest ]; then
```

Comment:
for external  this kind of check is in  `prepare_project`  I think it would be better to have there for this too, also because if prepare_project evolves the decision and the effect are closer

Reply:
agree, change done, thanks for reviewing